stix_shifter/stix_translation/src/modules/security_advisor/README.md

```diff
@@ -0,0 +1,81 @@
+# Security Advisor
+
+
+## Translating Stix Pattern to Security Advisor Query
+
+```
+python3 main.py translate "security_advisor" "query" '{}' "[url:value = 'http://5.188.86.29:7000' OR url:value = 'http://5.45.69.149:7000']"
+```
+
+`{'queries': [["[url:value = 'http://5.188.86.29:7000' OR url:value = 'http://5.45.69.149:7000']"]]}`
```

Comment:
The inner-most list, is this something cloud security advisor expects as part of the query? Regarding the two outer most lists, stix-shifter should only be returning one list of one or more queries, this nesting is going to break things.

Reply:
Handled this as we discussed

Reply:
I see that. You'll want to update this line of the readme to reflect the change.